### DISCUSSION TITLE
`uds` support in proxy classes

### DISCUSSION THREAD

**Discussion by NewUserHa** [encode/httpcore#858] [Potential Issue]
related to https://github.com/encode/httpcore/pull/857
currently, `AsyncConnectionPool` supported `uds` but the proxy classes(socksproxy) inherited from it does not

**Comment by lovelydinosaur** [comment]
Is making a unix domain socket connection to a proxy a use-case anyone's bumped up against yet?

**Reply by NewUserHa** [reply]
It probably is an edge use case, but with the current httpcore design it is very simple to support.
